src/modules/Layout/Header/Header.js

```diff
@@ -7,7 +8,12 @@ import './Header.css';
 
 const b = block('j-header');
 
-const Header = ({ logged }) => {
+const propTypes = {
+    logged: PropTypes.bool.isRequired,
+    user: PropTypes.string.isRequired,
```

Comment:
maybe `username` fits this more.

Reply:
maybe in future, there will appear some additional info, so I will change `user<String>` to `user<Object>` and fill it with all necessary info. What do you think, @Vxplus ?

Reply:
Fine by me.